pytest.ini

```diff
@@ -0,0 +1,10 @@
+[pytest]
+# RuntimeWarning is due to scipy (or another package) that was compiled against an older numpy than is installed.
+# UserWarning is due to statement: tensorflow.compat.v2.io import gfile
+# DeprecationWarning is due to statement: tensorflow.compat.v2.io import gfile
```

Comment:
Should we be actually importing gfile in a differnet way?

Reply:
(to avoid this warning)

Reply:
Good suggestion. I didn't try it, but let me give it a shot. Thanks!